src/lib_json/json_value.cpp

```diff
@@ -1049,8 +1049,8 @@ bool Value::removeIndex(ArrayIndex index, Value* removed) {
   ArrayIndex oldSize = size();
   // shift left all items left, into the place of the "removed"
   for (ArrayIndex i = index; i < (oldSize - 1); ++i){
-    CZString key(i);
-    (*value_.map_)[key] = (*this)[i + 1];
+    CZString indexKey(i);
```

Comment:
did they change `key` to `indexKey`??

Reply:
Nope, I just renamed it from `key` to `indexKey` so it wouldn't shadow the other `key` variable.

Reply:
okay.  it was not obvious why this change was needed.